src/GitUtilities.js

```diff
@@ -86,6 +97,11 @@ export default class GitUtilities {
   }
 
   @logger.logifySync()
+  static getCurrentBranchDescription() {
+    return ChildProcessUtilities.execSync("git branch | sed -n '/\* /s///p'");
```

Comment:
This should be `git symbolic-ref --short -q HEAD`, not `sed` shenanigans. Thus, when an empty string, it represents a detached head state.